domains/event-store/poc/EventStore/FlatFile/FlatFileEventConsumer.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using EventStore;
+using EventStore.Serialization;
+using Topics;
+using EnergyOriginDateTimeExtension;
+
+namespace EventStore.Flatfile;
+
+public class FlatFileEventConsumer<T> : IDisposable, IEventConsumer<T> where T : EventModel {
+    long fromDate;
+    string topicSuffix;
+    string eventSuffix;
+    FileSystemWatcher rootWatcher;
+    List<FileSystemWatcher> watchers;
+    Queue<T> queue = new Queue<T>();
```

Comment:
Using https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentqueue-1?view=net-6.0 will help wrt. thread-safety - but I am pretty sure thread-safety has not been the main concern for the POC.